src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/changes/PrincipalAddressChange.java

```diff
@@ -6,26 +6,14 @@
 public class PrincipalAddressChange extends Change {
     private static final String CHANGE_NAME = "changeOfRoa";
 
-    @JsonProperty("registeredOfficeAddress")
-    private Address registeredOfficeAddress;
-
     @JsonProperty("proposedRegisteredOfficeAddress")
     private Address proposedRegisteredOfficeAddress;
 
-    public PrincipalAddressChange(Address registeredOfficeAddress, Address proposedRegisteredOfficeAddress){
+    public PrincipalAddressChange(Address proposedRegisteredOfficeAddress){
```

Comment:
Are any of the properties within the Address object required?

Reply:
Done